src/docs/sphinx/style_guide.rst

```diff
@@ -42,10 +42,9 @@ can come in handy to provide additional information:
      * Calculate du_dt = M^-1 (-Ku + f).
      * This is all that is needed for explicit methods
      * @param[in] u The state vector (input to the differentiation)
-     * @param[out] du The derivative of \p u with respect to time
+     * @param[out] du The derivative of @p u with respect to time
```

Comment:
was this change intentional?

Reply:
It's a style change - using the `@` instead of the `\` to indicate Doxygen commands

Reply:
This was probably a find/replace boo boo.